src/value.cpp

```diff
@@ -1608,6 +1608,11 @@ MemRef MemRef::subview(const vector<Expr> &offsets,
   }
 }
 
+MemRef MemRef::reshape(const std::vector<smt::Expr> &newDims) {
+  return MemRef(m, elemType, bid, offset,
+    newDims, MemRef::Layout(newDims), Expr::mkBool(true));
```

Comment:
Let's leave a comment in its declaration (value.h) saying that the returned MemRef is a view reference.

Reply:
resolved